components/admin_console/admin_navbar_dropdown/admin_navbar_dropdown.tsx

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage, injectIntl} from 'react-intl';
+
+import {Team} from 'mattermost-redux/types/teams';
+
+import * as GlobalActions from 'actions/global_actions.jsx';
+
+import {filterAndSortTeamsByDisplayName} from 'utils/team_utils.jsx';
+import {ModalIdentifiers} from 'utils/constants';
+import {intlShape} from 'utils/react_intl';
+
+import AboutBuildModal from 'components/about_build_modal';
+
+import Menu from 'components/widgets/menu/menu';
+
+import MenuItemBlockableLink from './menu_item_blockable_link';
+
+interface AdminNavbarDropdownProps {
+    intl: intlShape;
+    locale: string;
+    siteName?: string;
+    navigationBlocked?: boolean;
+    teams: Team[];
+    actions: {
+        deferNavigation?: (onNavigationConfirmed: any) => any;
```

Comment:
defined in admin_actions

Reply:
Actually, this is not optional, you can remove the `?` here.